### ISSUE TITLE
Leo inserts ~/.leo into sys.path but with wrong separators

### ISSUE THREAD

**Issue by tbpassin** [leo-editor/leo-editor#3231]
See PR #3233.

On Windows 10, my `sys.path` starts with an entry with the wrong path separators for Windows: `C:/Users/tom/.leo`.  

```
Leo 6.7.3-devel, devel branch, build fc31c98df4
2023-03-29 02:23:54 -0500
Python 3.10.9, PyQt version 6.4.1
Windows 10 AMD64 (build 10.0.19044) SP0
```
This could cause modules (like plugins) specified relative to `%USERPROFILE%/.leo` not to be found.

**Comment by edreamleo** [comment]
@tbpassin Thanks for this report. I'll fix it next.

**Comment by edreamleo** [comment]
Closed via PR #3233.
